### PR TITLE
HP-133

### PR DESCRIPTION
**Pull request by Audiosutras**
https://caktus.atlassian.net/browse/HP-133

### CODE REVIEW

File: hip/static/styles/pages/data_report_detail_page.scss [hunk 1]
```diff
@@ -1,6 +1,5 @@
 @import "../mixins";
 
-.reports-detail-page-mainarea-hip {
-  @include headers;
-  @include paragraphs;
+.reports-detail-page-hip {
```
Comment (dchukhin) [inline hip/static/styles/pages/data_report_detail_page.scss:3]: It looks like `reports-detail-page-hip` is not used in any HTML template, so these styles aren't applied on the report detail page

File: hip/static/styles/pages/data_report_archive_detail_page.scss [hunk 2]
```diff
@@ -1,6 +1,5 @@
 @import "../mixins";
 
-.reports-archive-detail-page-mainarea-hip {
-  @include headers;
-  @include paragraphs;
+.reports-archive-detail-page-hip {
```
Comment (dchukhin) [inline hip/static/styles/pages/data_report_archive_detail_page.scss:3]: It looks like `reports-archive-detail-page-hip` is not used in any HTML template, so these styles aren't applied on the report archive detail page

**Review by dchukhin — COMMENTED** [review]
Looks good for most of the files affected. After fixing the report ones, it should be ready to ➡️

File: hip/static/styles/pages/data_report_detail_page.scss (hunk 1 above)
Comment (Audiosutras) [reply hip/static/styles/pages/data_report_detail_page.scss:3]: Looks like I overwrote this class when I was copying and pasting. Great catch.

Comment (Audiosutras) [reply hip/static/styles/pages/data_report_detail_page.scss]: DONE

File: hip/static/styles/pages/data_report_archive_detail_page.scss (hunk 2 above)
Comment (Audiosutras) [reply hip/static/styles/pages/data_report_archive_detail_page.scss:3]: FIXED

**Comment by Audiosutras** [comment]
Now excluding breadcrumb from list mixin.

**Review by dchukhin — APPROVED** [review]
📃

**Pull request merged**